DevSkim-DotNet/Microsoft.DevSkim/resources/comments.json

```diff
@@ -48,31 +48,31 @@
       "sql"
     ],
     "inline": "--",
-    "preffix": "--",
+    "prefix": "--",
     "suffix": "\n"
   },
   {
     "language": [
       "clojure"
     ],
     "inline": ";;",
-    "preffix": ";;",
+    "prefix": ";;",
     "suffix": ""
   },
   {
     "language": [
       "vb"
     ],
     "inline": "'",
-    "preffix": "'",
+    "prefix": "'",
```

Comment:
Good catch. This was a legacy quirk. The JSON tag was misspelled in previous versions of DevSkim but spelled correctly in ApplicationInspector (which was originally forked from DevSkim). When I switched to using AI as the backend I neglected to update these.

Old Code that uses "preffix": https://github.com/microsoft/DevSkim/blob/a6d34984bd4695033a19598202c2c885d468045b/DevSkim-DotNet/Microsoft.DevSkim/Comment.cs#L18